docs/codeql/codeql-overview/codeql-changelog/codeql-cli-2.20.6.rst

```diff
@@ -71,6 +71,11 @@ Language Libraries
 Major Analysis Improvements
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Golang
```

Comment:
This should not be moved - looking at the blog, Go 1.24 support was added in 2.20.5, we just added the changenote later.